### DISCUSSION TITLE
How do I use "vae" in the "fooocus" program and with explanation ?

### DISCUSSION THREAD

**Discussion by 7HRAMZI** [lllyasviel/Fooocus#2647] [Q&A]
Welcome...

How do I use "vae" in "fooocus"?

How do I use "vae" in the "fooocus" program and with explanation ?

I'm really confused. I wanted to use "Animagine XL V3.1" but I found a written note (This checkpoint recommends a VAE, download and place it in the VAE folder.), download and place it in the VAE folder.) I did not find an explanation on YouTube explaining how to use “VAE” in “fooocus”.

Usually, I use “Fooocus\Fooocus\models\checkpoints\animaPencilXL_v100.safetensors” alone without downloading another “SDXL 1.0” model.

I want to explain the steps and a simplified explanation so that I can understand them. “I find it difficult to understand and sometimes I get confused in the steps.”
If possible, the explanation can be in a picture with numbering or a video.

I know my request is annoying, I apologize in advance if I upset you guys

I used Google Translate to send a question: “Sorry if my message is unclear or contains errors.

**Comment by Madxis** [comment]
It's not possible to use a different VAE than the one Fooocus has integrated, I don't know if they will change it in the future. I hope so because there are checkpoints that don't work well without their respective VAE.
But don't worry with Animagine XL V3.1, I tested it and it works fine with Fooocus built-in VAE.

**Reply by 7HRAMZI** [reply]
Thank you very much.

**Reply by mashb1t** [reply]
this is not true, you just can't change the VAE in the UI. Feel free to exchange the file with any VAE you like.

**Reply by Madxis** [reply]
Does this mean that if the VAE of the folder is replaced it would work?

**Reply by Sp0rky** [reply]
i can confirm this does not work, no amount of finagling with the VAE in the folder(s) or settings will override the default VAE baked into fooocus.

Only solution I found was to bake the VAE into the checkpoint itself, only then did it apply properly

**Reply by Mark0z1** [reply]
> Animagine

I can't find that one in the version I have :/
